iotcore/src/main/java/com/google/android/things/iotcore/IotCoreClient.java

```diff
@@ -232,16 +244,27 @@ private IotCoreClient(
         mClientConnectionState = clientConnectionState;
 
         mMqttClient.setCallback(
-                createMqttCallback(onConfigurationExecutor, onConfigurationListener));
+                createMqttCallback(onConfigurationExecutor, onConfigurationListener, onCommandExecutor, onCommandListener));
+
+        // Always subscribe to the device configuration topic
+        mSubscriptionTopics.add(mConnectionParams.getConfigurationTopic());
+
+        if (onCommandListener != null && onCommandExecutor != null) {
```

Comment:
don't we need to check the validity of onCommandListner and onCommandExecutor at line 233, just like configuration?

Reply:
yes, absolutely.